pinc/codepoint_validator.inc

```diff
@@ -18,8 +19,10 @@ $bad_char_message
 END;
 }
 
-function get_json_codepoints($projectid)
+function get_json_characters($projectid)
```

Comment:
we don't need this.

Reply:
Changed back, sans the `json_encode()`ing.